### ISSUE TITLE
Add historical and future spending 

### ISSUE THREAD

**Issue by Lorestan00** [danilkinkin/buckwheat#40]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when 

At the moment you can only add in expenses for today and not for the past or the future. 

**Describe the solution you'd like**

Have a calendar button to allow for future and past spends to be added.

**Comment by franco27** [comment]
Yes, it would be a good idea to include this functionality to add an expense for a future or past date.

It would also be interesting to automatically incorporate monthly charges. 
For example, a section in the settings with the automatic monthly withdrawal date for each expense. 
For example, 
Netflix is automatically withdrawn on the 9th of each month, 
mobile subscription on the 15th of the month...

This way, as the month progresses, the app can keep track of what expenses are still pending and show what is left to pay on the current day.

**Comment by shebangmed** [comment]
I support this. At this moment I add my future expenses the first day each period so that they get substracted from the remaining. I assume that the very first day of each period is fake, but this patches the lack of this feature.
